fontbe/src/features/marks.rs

```diff
@@ -776,16 +781,41 @@ mod tests {
         }
     }
 
+    // when describing an anchor position we almost always use integers, but
+    // for some tests we want to accept floats so we can verify rounding behaviour.
+    // we define our own trait and impl it for only these two types so that the
+    // compiler doesn't need additional type info when we use int/float literals.
```

Comment:
This makes me a little uncomfortable but I guess it's not public

Reply:
yes this is just for a test helper.